src/help.cpp

```diff
@@ -251,6 +251,14 @@ void add_options() {
   opt->help = "Number of initial records in table";
   opt->setInt(1000);
 
+  /* Initial Records in table */
+  opt = newOption(Option::BOOL, Option::EXACT_INITIAL_RECORDS,
+                  "exact-initial-records");
+  opt->help = "Insert exact number of initial records mention with --records. "
```

Comment:
opt->help = "--exact-initial-records when passed with --records option inserts"
                   " exact number of records in tables"